rasa/core/channels/socketio.py

```diff
@@ -13,8 +13,6 @@
 
 logger = logging.getLogger(__name__)
 
-CHAT_TEMPLATE_PATH = "/chat.html"
-
 
 class SocketBlueprint(Blueprint):
```

Comment:
`session_persistance` needs to be set to default to `False` again

Reply:
![Bildschirmfoto 2023-09-21 um 09 19 00](https://github.com/RasaHQ/rasa/assets/1098412/03d9bb63-016f-48b2-a7a5-539fb2c4be22)